reo/src/ghp.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+import pandas as pd
+import copy
+from reo.utilities import MMBTU_TO_KWH, TONHOUR_TO_KWHT
+from reo.src.incentives import Incentives, IncentivesNoProdBased
+from reo.src.data_manager import big_number
+
+
+class GHPGHXInputs:
+    """
+    Map the GHPGHX inputs to the objects in this class
+    """
+
+    ground_k_by_climate_zone = {
```

Comment:
It looks like the only use of the GHPGHXInputs class is the `ground_k_by_climate_zone` attribute. You have a TODO below about adding all of the input fields. Maybe we can settle on one input model? We could start by moving `ground_k_by_climate_zone` to the ghpghx/models.py perhaps.

Reply:
I moved this data to the /ghpghx endpoint. The /ground_conductivity endpoint is now served in a /ghpghx app view.